messages/run-dfa.js

```diff
@@ -20,6 +20,8 @@ module.exports = {
 		"rulethreadtimeoutDescriptionLong": "Time limit for evaluating a single entrypoint. Value in milliseconds. Inherits from SFGE_RULE_THREAD_TIMEOUT env-var if set. Default is 900,000 ms, or 15 minutes.",
 		"sevthresholdDescription": "throws an error when violations of specific severity (or more severe) are detected, invokes --normalize-severity",
 		"sevthresholdDescriptionLong": "Throws an error if violations are found with equal or greater severity than provided value. Values are 1 (high), 2 (moderate), and 3 (low). Exit code is the most severe violation. Using this flag also invokes the --normalize-severity flag",
+		"sfgejvmargsDescription": "JVM args to optimize SFGE execution",
+		"sfgejvmargsDescriptionLong": "Specify JVM args to optimize memory utilization of SFGE execution. TODO: more information on what params to include.",
```

Comment:
Will fix the message

Reply:
@rmohan20 Thanks for adding this to the UI text review doc. Since we're reviewing all UI text there, let's continue the conversation in the G doc.